packages/appstore/src/components/wallet-header/wallet-header.scss

```diff
@@ -78,12 +77,42 @@
                 }
 
                 &-text {
-                    display: none;
                     margin-left: 0.8rem;
                 }
 
-                &-active {
-                    display: block;
+                &-transition {
+                    &-enter {
+                        transform: translateX(-1rem);
+                        opacity: 0;
+                    }
+
+                    &-enter-active {
+                        transition: all 250ms ease-in-out;
```

Comment:
@sergei-deriv This should be 240 due to this message
https://deriv-group.slack.com/archives/C04ST0NNY5B/p1684201332274039?thread_ts=1683722945.985319&cid=C04ST0NNY5B

Reply:
@mahdiyeh-deriv I thought 10ms doesn't make difference and I have checked other CSSTransition in our app and it had 100, 150, 200, 250 and 500 ms, so I decided to use 250 instead of 240 :) But ok, I'll change it soon